### PR TITLE
python@3.11.5: Fix hash

### LINKED ISSUE #102: python@3.11.5: hash check failed
Checking hash of uninstall-pep-514.reg ... ERROR Hash check failed!
App:         business/python
URL:         https://raw.githubusercontent.com/ScoopInstaller/Main/master/scripts/python/uninstall-pep-514.reg
First bytes: 57 69 6E 64 6F 77 73 20
Expected:    d9309423b693ed63aea6b1fbfc0c34f16842cdca22a1b3edef283b87567a53b9
Actual:      65a46d5a9fec455b6fbc5d75f8a57cd5ee958d31873ef438766866a53759e9f6

### PR DESCRIPTION
**Pull request by BotServiceAutomation**
- Closes #102

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### python

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by alexchapar — APPROVED** [review]

**Pull request merged**
